.travis.yml

```diff
@@ -65,11 +65,6 @@ jobs:
         skip_cleanup: true
         on:
           all_branches: true
-    # push to dockerhub
```

Comment:
any specific reason to move it to drone?

Reply:
Yes, for the moment we're not doing CD with Travis, hence the move

Reply:
Yes, if we kept build in Travis, drone would have to wait for the image to be built. I don't like that dependency, I see no good things coming from that strategy